docs/source/howto/plugins.rst

```diff
@@ -0,0 +1,41 @@
+.. _how-to:plugins:
+
+*****************************
+How to write a plugin package
+*****************************
+
+
+.. _how-to:plugins:register:
+
+How to register new plugins
+===========================
+
+
+.. _how-to:plugins:bundle:
+
+How to bundle plugins in a package
+==================================
+
+
+.. _how-to:plugins:develop:
+
+How to develop a plugin package
+===============================
+
+
+.. _how-to:plugins:publish:
+
+How to publish a plugin package
+===============================
+
+
+.. _how-to:plugins:scheduler:
```

Comment:
Maybe you want to put these in a subsection in specific types of plugins? Otherwise it's a bit weird that we isolate only these two out of all the other types of plugins we have?

Reply:
yeah it's a good point, it is a bit awkward, but I am not sure where else to put it. I thought it was not a good idea to have a category "How to write a plugin" with individual sections for each possible plugin category, because that is not in the spirit of the how-to. The goal is not to write a plugin, the goal is to do something, and then we tell them in order to do that, what plugin they need to write. So the `CalcJob` manual will be in "How to run external codes".

Thinking about it now, since transport and scheduler really only applies to computers, which in itself only really applies (in most cases) to calculation jobs, maybe I can put these two sections at the end of the "How to run external codes" category? What do you think?

Reply:
These have been added to the end of "How to run external codes" since that is where they are really needed logically